test/built-ins/Set/prototype/add/add.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 23.2.3.1
+description: >
+    Set.prototype.add ( value )
+
+    17 ECMAScript Standard Built-in Objects
+
+includes: [propertyHelper.js]
+---*/
+
+assert.sameValue(
```

Comment:
Maybe we should have helper for validating standard methods.

We should also validate the `name` and `length` properties and that they do not have a `prototype` property.

```js
verifyBuiltinMethod(Set.prototype, 'add', 1)
```